src/core/server/licensing/licensing_config.ts

```diff
@@ -0,0 +1,36 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { Env } from '../config';
+import { LicensingConfigType } from './types';
+
+export class LicensingConfig {
+  public enabled: boolean;
+  public clusterSource: string;
+  public pollingFrequency: number;
+
+  /**
+   * @internal
+   */
+  constructor(rawConfig: LicensingConfigType, env: Env) {
+    this.enabled = rawConfig.enabled;
```

Comment:
I see `isEnabled` used in some places and `enabled` in others. Should be standardize on one or the other? My preference would be `isEnabled`, because I'm used to seeing words like `is` used to identify booleans.

Reply:
Since this comes from configuration that we control, this is a good idea. Will revise.